src/nupic/algorithms/Anomaly.cpp

```diff
@@ -109,10 +108,17 @@ Real32 Anomaly::compute(
   {
     score = (score >= binaryThreshold_) ? 1.0 : 0.0;
   }
+  // round result (to avoid near zero rounding errors)
+  score = roundf(score*(1/PRECISION))/PRECISION;
```

Comment:
@breznak @scottpurdy Here ya' go? http://stackoverflow.com/questions/4798777/is-there-a-c-equivalent-to-javas-bigdecimal